docs/Сети связи/Web/CSS.md

```diff
@@ -0,0 +1,24 @@
+#���� ���������������� CSS
+**CSS (Cascading Style Sheets)** - ��� ���� ����������������, ������� ������������ ��� �������� �������� ���� ���-�������.
+�� ����������, ��� �������� �� �������� ������ ���� ������������, ������� �����, ������, �������, ������� � ������ ������.
```

Comment:
[ch-md-error-separate-blocks](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-separate-blocks)